src/api/server-routes.ts

```diff
@@ -51,19 +51,19 @@ export function buildRoutes(): express.Router {
   const router = express.Router();
 
   router.get('/limit/:limit/offset/:offset', (req: express.Request, res: express.Response) => {
-    getBuilds(req.params.limit, req.params.offset).then(builds => {
+    getBuilds(req.params.limit, req.params.offset, req.query.userId).then(builds => {
       return res.status(200).json({ data: builds });
     });
   });
 
   router.get('/last', (req: express.Request, res: express.Response) => {
```

Comment:
please make nicer URLs, with conditional parameter `userId`. query parameters are not so nice.